### PR TITLE
Stripe integration with the project

### PR DESCRIPTION
**Pull request by swarnjit**
Fund project button will take the user to new page where funds can be added to the that project .

### CODE REVIEW

**Review by russ430 — APPROVED** [review]

File: client/src/App.js [hunk 1]
```diff
@@ -42,23 +46,31 @@ function App() {
...
-            <Redirect exact from="/" to="/login" />
-            <Route path="/explore" exact component={Explore} />
-            <Route path="/login" exact component={Login} />
-            <Route path="/signup" exact component={SignUp} />
-            <Route path="/users/:userId" component={Profile} />
-            <ProtectedRoutes path="/launch" exact component={Launch} />
-            <Route path="*" component={() => 'Page not found'} />
-          </Switch>
-        )}
+        <Elements stripe={stripePromise}>
```
Comment (skassam21) [inline client/src/App.js:49]: We probably want this to be only in the component that we are using it for (future reference)

File: client/src/components/Funding/Payments.js [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  const history = useHistory();
+  const stripe = useStripe();
+  const elements = useElements();
+
+  const handleCardDetailsChange = (e) => {
+    e.error ? setCheckoutError(e.error.message) : setCheckoutError();
+  };
+  const onSuccessfulCheckout = () => {
+    alert('Success');
+    history.push('/explore');
```
Comment (skassam21) [inline client/src/components/Funding/Payments.js:57]: Eventually we are also going to need to update the dashboard that the project was funded at this point

**Review by skassam21 — COMMENTED** [review]
Just some notes for future

File: client/src/App.js (hunk 1 above)
Comment (swarnjit) [reply client/src/App.js:49]: In Strip documentation they suggest it to use it at the root of the application to detect fraudulent attempts anywhere in the whole application. So thats why i used it here.

**Pull request merged**